receiver/mongodbatlasreceiver/metadata.yaml

```diff
@@ -0,0 +1,254 @@
+name: mongoatlasreceiver
+
+metrics:
+  process.asserts:
+    description: Number of assertions
+    unit: 1
+    data:
+      type: gauge
```

Comment:
All of these metrics are coming from MongoDB pre-aggregated- in this particular case, it's actually an asserts/second metric rather than a simple counter. I've gone through and fleshed out the units and documented which MongoDB metrics are associated with each OTel metric.